AAS V3.0 Classes and Attribute Identifiers (#31)
admin-shell-io/id: aas/3/0/README.md

# AAS (Asset Administration Shell)

> This is the sub-namespace reserved for the Asset Administration Shell Metamodel version 3.0.

Reviewer: ```suggestion
This is the sub-namespace reserved for the Asset Administration Shell Metamodel version 3.0 (IDTA-01001-3-0).
```

## Introduction

> The Metamodel itself is defined in the 'Details of the Asset Administration Shell - Part 1'. Only classes, attributes and named entities used in this model are allowed in this sub-namespace. The Metamodel is the core resource for the structure of the Asset Administration Shell and provides the frame for all further specifications. The following entities are defined in the Metamodel but also serve as an example how to use the 'Asset Administration Shell Identifiers' in general.

Reviewer: ```suggestion
The Metamodel itself is defined in the 'IDTA-01001-3-0 Specification of the Asset Administration Shell - Part 1: Metamodel'. Only classes, attributes and named entities used in this model are allowed in this sub-namespace. The Metamodel is the core resource for the structure of the Asset Administration Shell and provides the frame for all further specifications. The following entities are defined in the Metamodel but also serve as an example how to use the 'Asset Administration Shell Identifiers' in general.
```

## Status: Accepted

> The sub-namespace AAS and its identifiers have been accepted by the Coordination Board.

Reviewer: ```suggestion
The sub-namespace AAS and its identifiers have been accepted by the Coordination Board.
The identifiers are derived following the grammar as defined in IDTA-01001-3-0. 
```

> 

Reviewer: ```suggestion
## AasReferableNonIdentifiables
 Enumeration of all referable elements within an asset administration shell. Contains IdentifiableElements

 [http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables](http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables) The AasReferableNonIdentifiables class as defined in IDTA-01001-3-0.

## AasReferableNonIdentifiables/ANNOTATED_RELATIONSHIP_ELEMENT

 [http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables/ANNOTATED_RELATIONSHIP_ELEMENT](http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables/ANNOTATED_RELATIONSHIP_ELEMENT) The AasReferableNonIdentifiables ANNOTATED_RELATIONSHIP_ELEMENT of the AasReferableNonIdentifiables class as defined in IDTA-01001-3-0.

## AasReferableNonIdentifiables/BASIC_EVENT_ELEMENT

 [http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables/BASIC_EVENT_ELEMENT](http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables/BASIC_EVENT_ELEMENT) The AasReferableNonIdentifiables BASIC_EVENT_ELEMENT of the AasReferableNonIdentifiables class as defined in IDTA-01001-3-0.

## AasReferableNonIdentifiables/BLOB

 [http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables/BLOB](http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables/BLOB) The AasReferableNonIdentifiables BLOB of the AasReferableNonIdentifiables class as defined in IDTA-01001-3-0.

## AasReferableNonIdentifiables/CAPABILITY

 [http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables/CAPABILITY](http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables/CAPABILITY) The AasReferableNonIdentifiables CAPABILITY of the AasReferableNonIdentifiables class as defined in IDTA-01001-3-0.

## AasReferableNonIdentifiables/DATA_ELEMENT

 [http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables/DATA_ELEMENT](http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables/DATA_ELEMENT) The AasReferableNonIdentifiables DATA_ELEMENT of the AasReferableNonIdentifiables class as defined in IDTA-01001-3-0.

## AasReferableNonIdentifiables/ENTITY

 [http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables/ENTITY](http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables/ENTITY) The AasReferableNonIdentifiables ENTITY of the AasReferableNonIdentifiables class as defined in IDTA-01001-3-0.
 

## AasReferableNonIdentifiables/EVENT_ELEMENT

 [http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables/EVENT_ELEMENT](http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables/EVENT_ELEM;ENT) The AasReferableNonIdentifiables EVENT_ELEMENT of the AasReferableNonIdentifiables class as defined in IDTA-01001-3-0.

## AasReferableNonIdentifiables/FILE

 [http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables/FILE](http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables/FILE) The AasReferableNonIdentifiables FILE of the AasReferableNonIdentifiables class as defined in IDTA-01001-3-0.

## ReferableElement/MULTI_LANGUAGE_PROPERTY
 Property with a value that can be provided in multiple languages.

 [http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables/MULTI_LANGUAGE_PROPERTY](http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables/MULTI_LANGUAGE_PROPERTY) The AasReferableNonIdentifiables MULTI_LANGUAGE_PROPERTY of the AasReferableNonIdentifiables class as defined in IDTA-01001-3-0.

## AasReferableNonIdentifiables/OPERATION

 [http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables/OPERATION](http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables/OPERATION) The AasReferableNonIdentifiables OPERATION of the AasReferableNonIdentifiables class as defined in IDTA-01001-3-0.

## AasReferableNonIdentifiables/PROPERTY

 [http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables/PROPERTY](http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables/PROPERTY) The AasReferableNonIdentifiables PROPERTY of the AasReferableNonIdentifiables class as defined in IDTA-01001-3-0.

## AasReferableNonIdentifiables/RANGE

 [http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables/RANGE](http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables/RANGE) The AasReferableNonIdentifiables RANGE of the AasReferableNonIdentifiables class as defined in IDTA-01001-3-0.

## AasReferableNonIdentifiables/REFERENCE_ELEMENT

 [http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables/REFERENCE_ELEMENT](http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables/REFERENCE_ELEMENT) The AasReferableNonIdentifiables REFERENCE_ELEMENT of the AasReferableNonIdentifiables class as defined in IDTA-01001-3-0.

## AasReferableNonIdentifiables/RELATIONSHIP_ELEMENT

 [http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables/RELATIONSHIPT_ELEMENT](http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables/RELATIONSHIPT_ELEMENT) The AasReferableNonIdentifiables RELATIONSHIPT_ELEMENT of the AasReferableNonIdentifiables class as defined in IDTA-01001-3-0.

## AasReferableNonIdentifiables/SUBMODEL_ELEMENT

 [http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables/SUBMODEL_ELEMENT](http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables/SUBMODEL_ELEMENT) The AasReferableNonIdentifiables SUBMODEL_ELEMENT of the AasReferableNonIdentifiables class as defined in IDTA-01001-3-0.

## AasReferableNonIdentifiables/SUBMODEL_ELEMENT_COLLECTION
 Collection of Submodel Elements

 [http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables/SUBMODEL_ELEMENT_COLLECTION](http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables/SUBMODEL_ELEMENT_COLLECTION) The AasReferableNonIdentifiables SUBMODEL_ELEMENT_COLLECTION of the AasReferableNonIdentifiables class as defined in IDTA-01001-3-0.
 
## AasReferableNonIdentifiables/SUBMODEL_ELEMENT_LIST
 List of set of Submodel Elements

 [http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables/SUBMODEL_ELEMENT_LIST](http://admin-shell.io/aas/3/0/AasReferableNonIdentifiables/SUBMODEL_ELEMENT_LIST) The AasReferableNonIdentifiables SUBMODEL_ELEMENT_LIST of the AasReferableNonIdentifiables class as defined in IDTA-01001-3-0.

```

> ## AdministrativeInformation
>  Every Identifiable may have administrative information. Administrative information includes for example  Information about the version of the element  Information about who created or who made the last change to the element  Information about the languages available in case the element contains text, for translating purposed also themmaster or default language may be definedIn the first version of the AAS metamodel only version information as defined by IEC 61360 is defined. In later versions additional attributes may be added.
> 
>  [http://admin-shell.io/aas/3/0/AdministrativeInformation](http://admin-shell.io/aas/3/0/AdministrativeInformation) The AdministrativeInformation class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.
> 
> ## AdministrativeInformation/revision
>  Revision of the element. Constraint AASd-005: A revision requires a version. This means, if there is no version there is no revision neither.
> 
>  [http://admin-shell.io/aas/3/0/AdministrativeInformation/revision](http://admin-shell.io/aas/3/0/AdministrativeInformation/revision) The attribute revision of the AdministrativeInformation class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.
> 
> ## AdministrativeInformation/version
>  Version of the element.
> 
>  [http://admin-shell.io/aas/3/0/AdministrativeInformation/version](http://admin-shell.io/aas/3/0/AdministrativeInformation/version) The attribute version of the AdministrativeInformation class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.
> 

Reviewer: ```suggestion
## AdministrativeInformation
 Every Identifiable may have administrative information. Administrative information includes for example  Information about the version of the element  Information about who created or who made the last change to the element  Information about the languages available in case the element contains text, for translating purposed also themmaster or default language may be definedIn the first version of the AAS metamodel only version information as defined by IEC 61360 is defined. In later versions additional attributes may be added.

 [http://admin-shell.io/aas/3/0/AdministrativeInformation](http://admin-shell.io/aas/3/0/AdministrativeInformation) The AdministrativeInformation class as defined in IDTA-01001-3-0.

## AdministrativeInformation/creator
  The subject ID of the subject responsible for making the element 

 [http://admin-shell.io/aas/3/0/AdministrativeInformation/creator](http://admin-shell.io/aas/3/0/AdministrativeInformation/creator) The attribute creator of the AdministrativeInformation class as defined in IDTA-01001-3-0.

## AdministrativeInformation/templateId
  Identifier of the template that guided the creation of the element

 [http://admin-shell.io/aas/3/0/AdministrativeInformation/creator](http://admin-shell.io/aas/3/0/AdministrativeInformation/templateId) The attribute templateId of the AdministrativeInformation class as defined in IDTA-01001-3-0.
 
## AdministrativeInformation/revision
 Revision of the element. Constraint AASd-005: A revision requires a version. This means, if there is no version there is no revision neither.

 [http://admin-shell.io/aas/3/0/AdministrativeInformation/revision](http://admin-shell.io/aas/3/0/AdministrativeInformation/revision) The attribute revision of the AdministrativeInformation class as defined in IDTA-01001-3-0.

## AdministrativeInformation/version
 Version of the element.

 [http://admin-shell.io/aas/3/0/AdministrativeInformation/version](http://admin-shell.io/aas/3/0/AdministrativeInformation/version) The attribute version of the AdministrativeInformation class as defined in IDTA-01001-3-0.

```

Reviewer: two new attribute: creator and templateId
update text of AdministrativeInforamtion

## Asset
 An Asset describes meta data of an asset that is represented by an AAS. The asset may either represent an asset type or an asset instance. The asset has a globally unique identifier plus – if needed – additional domain specific (proprietary) identifiers.

 [http://admin-shell.io/aas/3/0/Asset](http://admin-shell.io/aas/3/0/Asset) The Asset class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

## AssetInformation
 In AssetInformation identifying meta data of the asset that is represented by an AAS is defined. The asset may either represent an asset type or an asset instance. The asset has a globally unique identifier plus – if needed – additional domain specific (proprietary) identifiers. However, to support the corner case of very first phase of lifecycle where a stabilised/constant global asset identifier does not already exist, the corresponding attribute “globalAssetId” is optional.

 [https://admin-shell.io/aas/3/0/AssetInformation](https://admin-shell.io/aas/3/0/AssetInformation) The AssetInformation class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

## AssetInformation/assetKind

>  Denotes whether the Asset of of kind 'Type' or 'Instance'.

Reviewer: ```suggestion
 Denotes whether the Asset of of kind 'Type' or 'Instance' or of kind 'not applicable'.
```

 [https://admin-shell.io/aas/3/0/AssetInformation/assetKind](https://admin-shell.io/aas/3/0/AssetInformation/assetKind) The attribute assetKind of the AssetInformation class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

> 

Reviewer: ```suggestion
## AssetInformation/assetType
 In case AssetInformation/assetKind is applicable the AssetInformation/assetType is the asset ID of the type asset of the asset under consideration as identified by AssetInformation/globalAssetId.

 [https://admin-shell.io/aas/3/0/AssetInformation/assetType](https://admin-shell.io/aas/3/0/AssetInformation/assetType) The attribute assetType of the AssetInformation class as defined in IDTA-01001-3-0.
```

## AssetInformation/globalAssetId
 Reference to either an Asset object or a global reference to the asset the AAS is representing. This attribute is required as soon as the AAS is exchanged via partners in the life cycle of the asset. In a first phase of the life cycle the asset might not yet have a global id but already an internal identifier. The internal identifier would be modelled via “externalAssetId”.

 [https://admin-shell.io/aas/3/0/AssetInformation/globalAssetId](https://admin-shell.io/aas/3/0/AssetInformation/globalAssetId) The attribute globalAssetId of the AssetInformation class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

> ## AssetInformation/externalAssetId
>  Additional domain specific external, typically proprietary Identifier for the asset like e.g. serial number etc.
> 
>  [https://admin-shell.io/aas/3/0/AssetInformation/externalAssetId](https://admin-shell.io/aas/3/0/AssetInformation/externalAssetId) The attribute externalAssetId of the AssetInformation class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

Reviewer: ```suggestion
## AssetInformation/specificAssetId
 Additional domain specific external, typically proprietary Identifier for the asset like e.g. serial number etc.

 [https://admin-shell.io/aas/3/0/AssetInformation/specificAssetId](https://admin-shell.io/aas/3/0/AssetInformation/specificAssetId) The attribute specificAssetId of the AssetInformation class as defined in IDTA-01001-3-0.
```

## AssetInformation/billOfMaterial
 A reference to a Submodel that defines the bill of material of the asset represented by the AAS. This submodel contains a set of entities describing the material used to compose the composite I4.0 Component.

 [https://admin-shell.io/aas/3/0/AssetInformation/billOfMaterial](https://admin-shell.io/aas/3/0/AssetInformation/billOfMaterial) The attribute billOfMaterial of the AssetInformation class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

> ## AssetInformation/thumbnail
>  Thumbnail of the asset represented by the asset administration shell.
> 
>  [https://admin-shell.io/aas/3/0/AssetInformation/thumbnail](https://admin-shell.io/aas/3/0/AssetInformation/thumbnail) The attribute thumbnail of the AssetInformation class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.
> 

Reviewer: ```suggestion
## AssetInformation/defaultThumbnail
 Thumbnail of the asset represented by the asset administration shell.

 [https://admin-shell.io/aas/3/0/AssetInformation/defaultThumbnail](https://admin-shell.io/aas/3/0/AssetInformation/defaultThumbnail) The attribute thumbnail of the AssetInformation class as defined in IDTA-01001-3-0'.

```

 Describes the Administration Shell for Assets, Products, Components, e.g. Machines

 [http://admin-shell.io/aas/3/0/AssetAdministrationShell](http://admin-shell.io/aas/3/0/AssetAdministrationShell) The AssetAdministrationShell class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

## AssetAdministrationShell/assetInformation
 Meta information about the asset the AAS is representing.

 [https://admin-shell.io/aas/3/0/AssetAdministrationShell/assetInformation](https://admin-shell.io/aas/3/0/AssetAdministrationShell/assetInformation) The attribute assetInformation of the AssetAdministrationShell class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

## AssetAdministrationShell/derivedFrom
 This relation connects instances of AAS with their respective types. Refer to Asset Kind for further information of instance and type kinds.

 [http://admin-shell.io/aas/3/0/AssetAdministrationShell/derivedFrom](http://admin-shell.io/aas/3/0/AssetAdministrationShell/derivedFrom) The attribute derivedFrom of the AssetAdministrationShell class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

> ## AssetAdministrationShell/security
>  Definition of the security relevant aspects of the AAS.
> 
>  [http://admin-shell.io/aas/3/0/AssetAdministrationShell/security](http://admin-shell.io/aas/3/0/AssetAdministrationShell/security) The attribute security of the AssetAdministrationShell class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

Reviewer: ```suggestion

```

Reviewer: no seucrity any longer in V3.0

## AssetAdministrationShell/submodel
 Points from the Admin Shell to the Submodels that describe the Admin Shell of a given Asset

>  [http://admin-shell.io/aas/3/0/AssetAdministrationShell/submodel](http://admin-shell.io/aas/3/0/AssetAdministrationShell/submodel) The attribute submodel of the AssetAdministrationShell class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

Reviewer: ```suggestion
 [http://admin-shell.io/aas/3/0/AssetAdministrationShell/submodels](http://admin-shell.io/aas/3/0/AssetAdministrationShell/submodels) The attribute submodel of the AssetAdministrationShell class as defined in IDTA-01001-3-0.
```

Author: The "UML" says AssetAdministrationShell/submodel (https://admin-shell-io.github.io/aas-specs-antora/IDTA-01001/v3.1/spec-metamodel/core.html#aas_attributes), submodels is only in JSON. That's why I stick to the singular form.

> ## AssetAdministrationShell/view
>  Points to the differents views associated to the Administration Shell via the Submodels.
> 
>  [http://admin-shell.io/aas/3/0/AssetAdministrationShell/view](http://admin-shell.io/aas/3/0/AssetAdministrationShell/view) The attribute view of the AssetAdministrationShell class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.
> 

Reviewer: ```suggestion

```

## AssetKind
 Enumeration for denoting whether an element is a type or an instance.

 [http://admin-shell.io/aas/3/0/AssetKind](http://admin-shell.io/aas/3/0/AssetKind) The AssetKind class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

> ## AssetKind/INSTANCE

Reviewer: the speicficaion says "Instance" - should we define in grammar that enumeration values have capital letters always?

Author: No, that doesn't make sense. Whatever the specifications states shall also appear here.

 Concrete, clearly identifiable component of a certain type.

 [http://admin-shell.io/aas/3/0/AssetKind/INSTANCE](http://admin-shell.io/aas/3/0/AssetKind/INSTANCE) The NamedIndividual INSTANCE of the AssetKind class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

## AssetKind/TYPE
 hardware or software element which specifies the common attributes shared by all instances of the type.

 [http://admin-shell.io/aas/3/0/AssetKind/TYPE](http://admin-shell.io/aas/3/0/AssetKind/TYPE) The NamedIndividual TYPE of the AssetKind class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

## BasicEvent
 A basic event.

 [http://admin-shell.io/aas/3/0/BasicEvent](http://admin-shell.io/aas/3/0/BasicEvent) The BasicEvent class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

> ## BasicEvent/observed
>  Reference to the data or other elements that are being observed.
> 
>  [http://admin-shell.io/aas/3/0/BasicEvent/observed](http://admin-shell.io/aas/3/0/BasicEvent/observed) The attribute observed of the BasicEvent class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.
> 

Reviewer: ```suggestion
## BasicEventElement/observed
 Reference to the data or other elements that are being observed.

 [http://admin-shell.io/aas/3/0/BasicEventElement/observed](http://admin-shell.io/aas/3/0/BasicEventElement/observed) The attribute observed of the BasicEventElement class as defined in IDTA-01001-3-0'.

```

 The semantics of a property or other elements that may have a semantic description is defined by a concept description. The description of the concept should follow a standardized schema (realized as data specification template).

 [http://admin-shell.io/aas/3/0/ConceptDescription](http://admin-shell.io/aas/3/0/ConceptDescription) The ConceptDescription class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

## ConceptDescription/content
 Link from a ConceptDescription to its explaining DataSpecificationContent.

 [https://admin-shell.io/aas/3/0/ConceptDescription/content](https://admin-shell.io/aas/3/0/ConceptDescription/content) The attribute content of the ConceptDescription as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

## ConceptDescription/isCaseOf
 Global reference to an external definition the concept is compatible to or was derived from.

 [http://admin-shell.io/aas/3/0/ConceptDescription/isCaseOf](http://admin-shell.io/aas/3/0/ConceptDescription/isCaseOf) The attribute isCaseOf of the ConceptDescription class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

> ## Constraint
>  A constraint is used to further qualify an element.
> 
>  [http://admin-shell.io/aas/3/0/Constraint](http://admin-shell.io/aas/3/0/Constraint) The Constraint class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.
> 

Reviewer: ```suggestion

```

## DataTypeIEC61360/URL_IEC6360_DATATYPE

 [http://admin-shell.io/aas/3/0/DataTypeIEC61360/URL_IEC6360_DATATYPE](http://admin-shell.io/aas/3/0/DataTypeIEC61360/URL_IEC6360_DATATYPE) The DataTypeIEC61360 URL_IEC6360_DATATYPE of the DataTypeIEC61360 class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

## Entity
 An entity is a submodel element that is used to model entities.

 [http://admin-shell.io/aas/3/0/Entity](http://admin-shell.io/aas/3/0/Entity) The Entity class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

## Entity/entityType
 Describes whether the entity is a co-managed entity or a self-managed entity.

 [http://admin-shell.io/aas/3/0/Entity/entityType](http://admin-shell.io/aas/3/0/Entity/entityType) The attribute entityType of the Entity class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

> ## Entity/externalAssetId
>  Reference to an identifier key value pair representing an external identifier of the asset represented by the asset administration shell.
> 
>  [http://admin-shell.io/aas/3/0/Entity/externalAssetId](http://admin-shell.io/aas/3/0/Entity/externalAssetId) The attribute externalAssetId of the Entity class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.
> 

Reviewer: ```suggestion
## Entity/specificAssetId
 Reference to an identifier key value pair representing an external identifier of the asset represented by the asset administration shell.

 [http://admin-shell.io/aas/3/0/Entity/specificAssetId](http://admin-shell.io/aas/3/0/Entity/specificAssetId) The attribute specificAssetId of the Entity class as defined in IDTA-01001-3-0.

```

> ## Entity/globalAssetId

Reviewer: ordering

## EntityType/CO_MANAGED_ENTITY
 For co-managed entities there is no separate AAS. Co-managed entities need to be part of a self-managed entity.

 [http://admin-shell.io/aas/3/0/EntityType/CO_MANAGED_ENTITY](http://admin-shell.io/aas/3/0/EntityType/CO_MANAGED_ENTITY) The NamedIndividual CO_MANAGED_ENTITY of the EntityType class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

## EntityType/SELF_MANAGED_ENTITY
 Self-Managed Entities have their own AAS but can be part of the bill of material of a composite self-managed entity. The asset of an I4.0 Component is a self-managed entity per definition.

 [http://admin-shell.io/aas/3/0/EntityType/SELF_MANAGED_ENTITY](http://admin-shell.io/aas/3/0/EntityType/SELF_MANAGED_ENTITY) The NamedIndividual SELF_MANAGED_ENTITY of the EntityType class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.
## Extension
 Single extension of an element.

 [https://admin-shell.io/aas/3/0/Extension](https://admin-shell.io/aas/3/0/Extension) The Extension class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.
## Extension/name

>  An extension of the element.

Reviewer: ```suggestion
 Name of the extension.
```

 [https://admin-shell.io/aas/3/0/Extension/name](https://admin-shell.io/aas/3/0/Extension/name) The attribute name of the Extension class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

## Extension/valueType
 Type of the value of the extension.

 [https://admin-shell.io/aas/3/0/Extension/valueType](https://admin-shell.io/aas/3/0/Extension/valueType) The attribute valueType of the Extension class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

> ## Extension/value

Reviewer: ordering

 Value of the extension.

 [https://admin-shell.io/aas/3/0/Extension/value](https://admin-shell.io/aas/3/0/Extension/value) The attibute value of the Extension class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

## Extension/refersTo
 Reference to an element the extension refers to.

 [https://admin-shell.io/aas/3/0/Extension/refersTo](https://admin-shell.io/aas/3/0/Extension/refersTo) The attribute refersTo Extension class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

> ## Event
> 
> 
>  [http://admin-shell.io/aas/3/0/Event](http://admin-shell.io/aas/3/0/Event) The Event class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.
> 

Reviewer: ```suggestion
## EventElement
An event element.

 [http://admin-shell.io/aas/3/0/EventElement](http://admin-shell.io/aas/3/0/EventElement) The EventElement class as defined in Details of the IDTA-01001-3-0.

```

## EventElement
 Defines the necessary information for sending or receiving events.

 [http://admin-shell.io/aas/3/0/EventElement](http://admin-shell.io/aas/3/0/EventElement) The EventElement class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

## EventMessage
 Defines the necessary information ofv an event instance sent out or received.

 [http://admin-shell.io/aas/3/0/EventMessage](http://admin-shell.io/aas/3/0/EventMessage) The EventMessage class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

## File
 A File is a data element that represents a file via its path description.

 [http://admin-shell.io/aas/3/0/File](http://admin-shell.io/aas/3/0/File) The File class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

> ## File/mimeType
>  Mime type of the content of the File.
> 
>  [https://admin-shell.io/aas/3/0/File/mimeType](https://admin-shell.io/aas/3/0/File/mimeType) The attribute mimeType of the File class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.
> 

Reviewer: ```suggestion
## File/contentType
 Content type of the content of the File.

 [https://admin-shell.io/aas/3/0/File/contentType](https://admin-shell.io/aas/3/0/File/contentType) The attribute mimeType of the File class as defined in IDTA-01001-3-0.

```

## File/value
 Path and name of the referenced file (with file extension). The path can be absolute or relative.

 [https://admin-shell.io/aas/3/0/File/value](https://admin-shell.io/aas/3/0/File/value) The attribute value of the File class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

> ## Formula
> 
> 
>  [http://admin-shell.io/aas/3/0/Formula](http://admin-shell.io/aas/3/0/Formula) The Formula class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

Reviewer: ```suggestion
```

> 
> ## Formula/dependsOn
>  A formula may depend on referable or even external global elements - assumed that can be referenced and their value may be evaluated - that are used in the logical expression.
> 
>  [http://admin-shell.io/aas/3/0/Formula/dependsOn](http://admin-shell.io/aas/3/0/Formula/dependsOn) The attribute dependsOn of the Formula class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.
> 

Reviewer: ```suggestion

```

## HasKind/kind
 ModelingKind of the element: either type or instance.

 [http://admin-shell.io/aas/3/0/HasKind/kind](http://admin-shell.io/aas/3/0/HasKind/kind) The attribute kind of the HasKind class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

## HasSemantics
 Element that can have a semantic definition. Identifier of the semantic definition of the element. It is called semantic id of the element. The semantic id may either reference an external global id or it may reference a referable model element of kind=Type that defines the semantics of the element.

 [http://admin-shell.io/aas/3/0/HasSemantics](http://admin-shell.io/aas/3/0/HasSemantics) The HasSemantics class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

## HasSemantics/semanticId
 Points to the Expression Semantic of the Submodels

 [http://admin-shell.io/aas/3/0/HasSemantics/semanticId](http://admin-shell.io/aas/3/0/HasSemantics/semanticId) The attribute semanticId of the HasSemantics class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

> 

Reviewer: ```suggestion
## HasSemantics/supplementalSemanticId
 Identifier of a supplemental semantic definition of the element called supplemental semantic ID of the element

 [http://admin-shell.io/aas/3/0/HasSemantics/supplementalSemanticId](http://admin-shell.io/aas/3/0/HasSemantics/supplementalSemanticId) The attribute supplementalSemanticId of the HasSemantics class as defined in IDTA-01001-3-0.

```

## Identifiable
 An element that has a globally unique identifier.

 [http://admin-shell.io/aas/3/0/Identifiable](http://admin-shell.io/aas/3/0/Identifiable) The Identifiable class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

## Identifiable/administration
 Administrative information of an identifiable element.

 [http://admin-shell.io/aas/3/0/Identifiable/administration](http://admin-shell.io/aas/3/0/Identifiable/administration) The attribute administration of the Identifiable class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

> ## Identifiable/identification
>  The globally unique identification of the element.
> 
>  [http://admin-shell.io/aas/3/0/Identifiable/identification](http://admin-shell.io/aas/3/0/Identifiable/identification) The attribute identification of the Identifiable class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

Reviewer: ```suggestion
## Identifiable/id
 The globally unique identification of the element.

 [http://admin-shell.io/aas/3/0/Identifiable/id](http://admin-shell.io/aas/3/0/Identifiable/id) The attribute identification of the Identifiable class as defined in IDTA-01001-3-0.
```

> 
> ## IdentifiableElement
>  Enumeration of all identifiable elements within an asset administration shell that are not identifiable
> 
>  [http://admin-shell.io/aas/3/0/IdentifiableElement](http://admin-shell.io/aas/3/0/IdentifiableElement) The IdentifiableElement class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.
> 
> ## IdentifiableElement/ASSET
> 
> 
>  [http://admin-shell.io/aas/3/0/IdentifiableElement/ASSET](http://admin-shell.io/aas/3/0/IdentifiableElement/ASSET) The IdentifiableElement ASSET of the IdentifiableElement class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.
> 
> ## IdentifiableElement/ASSET_IDENTIFIABLE_ELEMENT
> 
> 
>  [http://admin-shell.io/aas/3/0/IdentifiableElement/ASSET_IDENTIFIABLE_ELEMENT](http://admin-shell.io/aas/3/0/IdentifiableElement/ASSET_IDENTIFIABLE_ELEMENT) The IdentifiableElement ASSET_IDENTIFIABLE_ELEMENT of the IdentifiableElement class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.
> 
> ## IdentifiableElement/CONCEPT_DESCRIPTION
> 
> 
>  [http://admin-shell.io/aas/3/0/IdentifiableElement/CONCEPT_DESCRIPTION](http://admin-shell.io/aas/3/0/IdentifiableElement/CONCEPT_DESCRIPTION) The IdentifiableElement CONCEPT_DESCRIPTION of the IdentifiableElement class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.
> 
> ## IdentifiableElement/SUBMODEL
> 
> 
>  [http://admin-shell.io/aas/3/0/IdentifiableElement/SUBMODEL](http://admin-shell.io/aas/3/0/IdentifiableElement/SUBMODEL) The IdentifiableElement SUBMODEL of the IdentifiableElement class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.
> 

Reviewer: ```suggestion

```

Reviewer: renamed to AasIdentifiables

## LevelType/TYP
 TYP according to IEC 61360 as an Identifier scheme for typical levels.

 [https://admin-shell.io/aas/3/0/LevelType/TYP](https://admin-shell.io/aas/3/0/LevelType/TYP) The TYP of the LevelType class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

## LocalKeyType
 Enumeration of different key value types within a key.

 [http://admin-shell.io/aas/3/0/LocalKeyType](http://admin-shell.io/aas/3/0/LocalKeyType) The LocalKeyType class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

## LocalKeyType/FRAGMENT_ID
 Identifier of a fragment within a file

 [http://admin-shell.io/aas/3/0/LocalKeyType/FRAGMENT_ID](http://admin-shell.io/aas/3/0/LocalKeyType/FRAGMENT_ID) The LocalKeyType FRAGMENT_ID of the LocalKeyType class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

## LocalKeyType/IDSHORT
 idShort of a referable element

 [http://admin-shell.io/aas/3/0/LocalKeyType/IDSHORT](http://admin-shell.io/aas/3/0/LocalKeyType/IDSHORT) The LocalKeyType IDSHORT of the LocalKeyType class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

> ## ModelingKind
>  Enumeration for denoting whether an element is a type or an instance.
> 
>  [http://admin-shell.io/aas/3/0/ModelingKind](http://admin-shell.io/aas/3/0/ModelingKind) The ModelingKind class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

Reviewer: ```suggestion
## ModellingKind
 Enumeration for denoting whether an element is a type or an instance.

 [http://admin-shell.io/aas/3/0/ModellingKind](http://admin-shell.io/aas/3/0/ModellingKind) The ModellingKind class as defined IDTA-01001-3-0.
```

## MultiLanguageProperty
 A property is a data element that has a multi language value.

 [http://admin-shell.io/aas/3/0/MultiLanguageProperty](http://admin-shell.io/aas/3/0/MultiLanguageProperty) The MultiLanguageProperty class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

## MultiLanguageProperty/value
 The value of the property instance.

 [https://admin-shell.io/aas/3/0/MultiLanguageProperty/value](https://admin-shell.io/aas/3/0/MultiLanguageProperty/value) The attribute value of the MultiLanguageProperty class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

## MultiLanguageProperty/valueId
 Reference to the global unqiue id of a coded value.

 [https://admin-shell.io/aas/3/0/MultiLanguageProperty/valueId](https://admin-shell.io/aas/3/0/MultiLanguageProperty/valueId) The attribute valueId of the MultiLanguageProperty class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

> ## ObjectAttributes
>  A set of data elements that describe object attributes. These attributes need to refer to a data element within an existing submodel.
> 
>  [http://admin-shell.io/aas/3/0/ObjectAttributes](http://admin-shell.io/aas/3/0/ObjectAttributes) The ObjectAttributes class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.
> 
> ## ObjectAttributes/objectAttribute
>  A data elements that further classifies an object.
> 
>  [http://admin-shell.io/aas/3/0/ObjectAttributes/objectAttribute](http://admin-shell.io/aas/3/0/ObjectAttributes/objectAttribute) The attribute objectAttribute of the ObjectAttributes class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.
> 

Reviewer: ```suggestion

```

## Operation/outputVariable
 Output parameter of the operation.

 [http://admin-shell.io/aas/3/0/Operation/outputVariable](http://admin-shell.io/aas/3/0/Operation/outputVariable) The attribute outputVariable of the Operation class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

## OperationVariable

 [http://admin-shell.io/aas/3/0/OperationVariable](http://admin-shell.io/aas/3/0/OperationVariable) The OperationVariable class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

## OperationVariable/value
 Describes the needed argument for an operation via a submodel element of kind=Template.

 [http://admin-shell.io/aas/3/0/OperationVariable/value](http://admin-shell.io/aas/3/0/OperationVariable/value) The attribute value of the OperationVariable class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

## Property
 A property is a data element that has a single value.

 [http://admin-shell.io/aas/3/0/Property](http://admin-shell.io/aas/3/0/Property) The Property class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

> ## Property/category
>  The following categories are defined for properties and multi-language properties: CONSTANT, PARAMETER, and VARIABLE.
> 
>  [https://admin-shell.io/aas/3/0/Property/category](https://admin-shell.io/aas/3/0/Property/category) The attribute category of the Property class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.
> 

Reviewer: ```suggestion

```

Reviewer: gab es so wohl noch nie? Immer geerbt von Referable

## Property/value
 The value of the property instance.

 [https://admin-shell.io/aas/3/0/Property/value](https://admin-shell.io/aas/3/0/Property/value) The attribute value of the Property class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

> ## Property/propertyValueId
>  Reference to the global unqiue id of a coded value.
> 
>  [http://admin-shell.io/aas/3/0/Property/propertyValueId](http://admin-shell.io/aas/3/0/Property/propertyValueId) The attribute propertyValueId of the Property class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.
> 

Reviewer: ```suggestion
## Property/propertyValueId
 Reference to the global unqiue id of a coded value.

 [http://admin-shell.io/aas/3/0/Property/valueId](http://admin-shell.io/aas/3/0/Property/valueId) The attribute valueId of the Property class as defined in IDTA-01001-3-0.

```

## Qualifiable
 Additional qualification of a qualifiable element.

 [http://admin-shell.io/aas/3/0/Qualifiable](http://admin-shell.io/aas/3/0/Qualifiable) The Qualifiable class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

## Qualifiable/qualifier
 Additional qualification of a qualifiable element.

 [http://admin-shell.io/aas/3/0/Qualifiable/qualifier](http://admin-shell.io/aas/3/0/Qualifiable/qualifier) The attribute qualifier of the Qualifiable class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

## Qualifier
 A qualifier is a type-value pair that makes additional statements w.r.t. the value of the element.

 [http://admin-shell.io/aas/3/0/Qualifier](http://admin-shell.io/aas/3/0/Qualifier) The Qualifier class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

> 

Reviewer: ```suggestion
## Qualifier/kind
 The qualifier kind describes the kind of qualifier that is applied to the element.

 [http://admin-shell.io/aas/3/0/Qualifier/kind](http://admin-shell.io/aas/3/0/Qualifier/kind) The attribute kind of the Qualifier class as defined in IDTA-01001-3-0.
```

## Qualifier/type
 The qualifier type describes the type of the qualifier that is applied to the element.

 [http://admin-shell.io/aas/3/0/Qualifier/type](http://admin-shell.io/aas/3/0/Qualifier/type) The attribute type of the Qualifier class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

## Qualifier/value
 The qualifier value is the value of the qualifier.

 [http://admin-shell.io/aas/3/0/Qualifier/value](http://admin-shell.io/aas/3/0/Qualifier/value) The attribute value of the Qualifier class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

## Qualifier/valueId
 Reference to the global unqiue id of a coded value.

 [http://admin-shell.io/aas/3/0/Qualifier/valueId](http://admin-shell.io/aas/3/0/Qualifier/valueId) The attribute valueId of the Qualifier class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

> 

Reviewer: # Qualifier/valueType
  Data type of the qualifier value.

 [http://admin-shell.io/aas/3/0/Qualifier/valueType](http://admin-shell.io/aas/3/0/Qualifier/valueType) The attribute valueType of the Qualifier class as defined in IDTA-01001-3-0.

> 

Reviewer: ```suggestion

# Qualifier/valueType
  Data type of the qualifier value.

 [http://admin-shell.io/aas/3/0/Qualifier/valueType](http://admin-shell.io/aas/3/0/Qualifier/valueType) The attribute valueType of the Qualifier class as defined in IDTA-01001-3-0.
 
```

## Range
 An element that is referable by its idShort. This id is not globally unique. This id is unique within the name space of the element.

 [http://admin-shell.io/aas/3/0/Range](http://admin-shell.io/aas/3/0/Range) The Range class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

## Range/max
 The maximum value of the range.

 [http://admin-shell.io/aas/3/0/Range/max](http://admin-shell.io/aas/3/0/Range/max) The attribute max of the Range class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

> ## Range/min

Reviewer: ordering

Author: I sorted it following the lexical order: "Range/max" < "Range/min"...

 The minimum value of the range.

 [http://admin-shell.io/aas/3/0/Range/min](http://admin-shell.io/aas/3/0/Range/min) The attribute min of the Range class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

> 

Reviewer: ```suggestion

## Range/valueType
 Data type of the min und max attributes

 [http://admin-shell.io/aas/3/0/Range/valueType](http://admin-shell.io/aas/3/0/Range/valueType) The attribute valueType of the Range class as defined in IDTA-01001-3-0.
 
```

## Referable/description
 Description or comments on the element. The description can be provided in several languages.

 [http://admin-shell.io/aas/3/0/Referable/description](http://admin-shell.io/aas/3/0/Referable/description) The attribute description of the Referable class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

## Referable/displayName
 Display name. Can be provided in several languages.

[https://admin-shell.io/aas/3/0/Referable/displayName](https://admin-shell.io/aas/3/0/Referable/displayName) The attribute displayName of the Referable class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

## Referable/idShort
 Identifying string of the element within its name space.

 [http://admin-shell.io/aas/3/0/Referable/idShort](http://admin-shell.io/aas/3/0/Referable/idShort) The attribute idShort of the Referable class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

> ## Referable/parent
>  Reference to the next referable parent element of the element. Constraint AASd-004: Add parent in case of non-identifiable elements.
> 
>  [http://admin-shell.io/aas/3/0/Referable/parent](http://admin-shell.io/aas/3/0/Referable/parent) The attribute parent of the Referable class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.
> 

Reviewer: ```suggestion

```

## ReferableElements/BASIC_EVENT

 [http://admin-shell.io/aas/3/0/ReferableElements/BASIC_EVENT](http://admin-shell.io/aas/3/0/ReferableElements/BASIC_EVENT) The ReferableElements BASIC_EVENTT of the ReferableElements class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

## ReferableElements/BLOB

 [http://admin-shell.io/aas/3/0/ReferableElements/BLOB](http://admin-shell.io/aas/3/0/ReferableElements/BLOB) The ReferableElements BLOB of the ReferableElements class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

## ReferableElements/CAPABILITY

 [http://admin-shell.io/aas/3/0/ReferableElements/CAPABILITY](http://admin-shell.io/aas/3/0/ReferableElements/CAPABILITY) The ReferableElements CAPABILITY of the ReferableElements class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

> ## ReferableElements/CONCEPT_DICTIONARY
> 
> 
>  [http://admin-shell.io/aas/3/0/ReferableElements/CONCEPT_DICTIONARY](http://admin-shell.io/aas/3/0/ReferableElements/CONCEPT_DICTIONARY) The ReferableElements CONCEPT_DICTIONARY of the ReferableElements class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.
> 
> ## ReferableElements/DATA_ELEMENT
> 
> 
>  [http://admin-shell.io/aas/3/0/ReferableElements/DATA_ELEMENT](http://admin-shell.io/aas/3/0/ReferableElements/DATA_ELEMENT) The ReferableElements DATA_ELEMENT of the ReferableElements class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.
> 
> ## ReferableElements/ENTITY
> 
> 
>  [http://admin-shell.io/aas/3/0/ReferableElements/ENTITY](http://admin-shell.io/aas/3/0/ReferableElements/ENTITY) The ReferableElements ENTITY of the ReferableElements class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.
> 
> ## ReferableElements/EVENT
> 
> 
>  [http://admin-shell.io/aas/3/0/ReferableElements/EVENT](http://admin-shell.io/aas/3/0/ReferableElements/EVENT) The ReferableElements EVENT of the ReferableElements class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.
> 
> ## ReferableElement/MULTI_LANGUAGE_PROPERTY
>  Property with a value that can be provided in multiple languages.
> 
>  [http://admin-shell.io/aas/3/0/ReferableElements/MULTI_LANGUAGE_PROPERTY](http://admin-shell.io/aas/3/0/ReferableElements/MULTI_LANGUAGE_PROPERTY) The ReferableElements MULTI_LANGUAGE_PROPERTY of the ReferableElements class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.
> 
> ## ReferableElements/OPERATION
> 
> 
>  [http://admin-shell.io/aas/3/0/ReferableElements/OPERATION](http://admin-shell.io/aas/3/0/ReferableElements/OPERATION) The ReferableElements OPERATION of the ReferableElements class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.
> 
> ## ReferableElements/PROPERTY
> 
> 
>  [http://admin-shell.io/aas/3/0/ReferableElements/PROPERTY](http://admin-shell.io/aas/3/0/ReferableElements/PROPERTY) The ReferableElements PROPERTY of the ReferableElements class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.
> 
> ## ReferableElements/RANGE
> 
> 
>  [http://admin-shell.io/aas/3/0/ReferableElements/RANGE](http://admin-shell.io/aas/3/0/ReferableElements/RANGE) The ReferableElements RANGE of the ReferableElements class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.
> 
> ## ReferableElements/REFERENCE_ELEMENT
> 
> 
>  [http://admin-shell.io/aas/3/0/ReferableElements/REFERENCE_ELEMENT](http://admin-shell.io/aas/3/0/ReferableElements/REFERENCE_ELEMENT) The ReferableElements REFERENCE_ELEMENT of the ReferableElements class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.
> 
> ## ReferableElements/RELATIONSHIPT_ELEMENT
> 
> 
>  [http://admin-shell.io/aas/3/0/ReferableElements/RELATIONSHIPT_ELEMENT](http://admin-shell.io/aas/3/0/ReferableElements/RELATIONSHIPT_ELEMENT) The ReferableElements RELATIONSHIPT_ELEMENT of the ReferableElements class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.
> 
> ## ReferableElements/SUBMODEL_ELEMENT_COLLECTION
>  Collection of Submodel Elements
> 
>  [http://admin-shell.io/aas/3/0/ReferableElements/SUBMODEL_ELEMENT_COLLECTION](http://admin-shell.io/aas/3/0/ReferableElements/SUBMODEL_ELEMENT_COLLECTION) The ReferableElements SUBMODEL_ELEMENT_COLLECTION of the ReferableElements class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.
> 
> ## ReferableElements/SUBMODEL_ELEMENT
> 
> 
>  [http://admin-shell.io/aas/3/0/ReferableElements/SUBMODEL_ELEMENT](http://admin-shell.io/aas/3/0/ReferableElements/SUBMODEL_ELEMENT) The ReferableElements SUBMODEL_ELEMENT of the ReferableElements class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.
> 

Reviewer: ```suggestion

## AasIdentifiables
 Enumeration of all referable elements within an asset administration shell. Contains IdentifiableElements

 [http://admin-shell.io/aas/3/0/AasIdentifiables](http://admin-shell.io/aas/3/0/AasIdentifiables) The AasIdentifiables class as defined in IDTA-01001-3-0.
 
## AasSubmodelElements
 Enumeration of all referable elements within an asset administration shell. Contains IdentifiableElements

 [http://admin-shell.io/aas/3/0/AasSubmodelElements](http://admin-shell.io/aas/3/0/AasSubmodelElements) The AasSubmodelElements class as defined in IDTA-01001-3-0.

## AasSubmodelElements/ANNOTATED_RELATIONSHIP_ELEMENT

 [http://admin-shell.io/aas/3/0/AasSubmodelElements/ANNOTATED_RELATIONSHIP_ELEMENT](http://admin-shell.io/aas/3/0/AasSubmodelElements/ANNOTATED_RELATIONSHIP_ELEMENT) The AasSubmodelElements ANNOTATED_RELATIONSHIP_ELEMENT of the AasSubmodelElements class as defined in IDTA-01001-3-0.

## AasSubmodelElements/BASIC_EVENT_ELEMENT

 [http://admin-shell.io/aas/3/0/AasSubmodelElements/BASIC_EVENT_ELEMENT](http://admin-shell.io/aas/3/0/AasSubmodelElements/BASIC_EVENT_ELEMENT) The AasSubmodelElements BASIC_EVENT_ELEMENT of the AasSubmodelElements class as defined in IDTA-01001-3-0.

## AasSubmodelElements/BLOB

 [http://admin-shell.io/aas/3/0/AasSubmodelElements/BLOB](http://admin-shell.io/aas/3/0/AasSubmodelElements/BLOB) The AasSubmodelElements BLOB of the AasSubmodelElements class as defined in IDTA-01001-3-0.

## AasSubmodelElements/CAPABILITY

 [http://admin-shell.io/aas/3/0/AasSubmodelElements/CAPABILITY](http://admin-shell.io/aas/3/0/AasSubmodelElements/CAPABILITY) The AasSubmodelElements CAPABILITY of the AasSubmodelElements class as defined in IDTA-01001-3-0.

## AasSubmodelElements/DATA_ELEMENT

 [http://admin-shell.io/aas/3/0/AasSubmodelElements/DATA_ELEMENT](http://admin-shell.io/aas/3/0/AasSubmodelElements/DATA_ELEMENT) The AasSubmodelElements DATA_ELEMENT of the AasSubmodelElements class as defined in IDTA-01001-3-0.

## AasSubmodelElements/ENTITY

 [http://admin-shell.io/aas/3/0/AasSubmodelElements/ENTITY](http://admin-shell.io/aas/3/0/AasSubmodelElements/ENTITY) The AasSubmodelElements ENTITY of the AasSubmodelElements class as defined in IDTA-01001-3-0.

## AasSubmodelElements/EVENT_ELEMENT

 [http://admin-shell.io/aas/3/0/AasSubmodelElements/EVENT_ELEMENT](http://admin-shell.io/aas/3/0/AasSubmodelElements/EVENT_ELEM;ENT) The AasSubmodelElements EVENT_ELEMENT of the AasSubmodelElements class as defined in IDTA-01001-3-0.

## ReferableElement/MULTI_LANGUAGE_PROPERTY
 Property with a value that can be provided in multiple languages.

 [http://admin-shell.io/aas/3/0/AasSubmodelElements/MULTI_LANGUAGE_PROPERTY](http://admin-shell.io/aas/3/0/AasSubmodelElements/MULTI_LANGUAGE_PROPERTY) The AasSubmodelElements MULTI_LANGUAGE_PROPERTY of the AasSubmodelElements class as defined in IDTA-01001-3-0.

## AasSubmodelElements/OPERATION

 [http://admin-shell.io/aas/3/0/AasSubmodelElements/OPERATION](http://admin-shell.io/aas/3/0/AasSubmodelElements/OPERATION) The AasSubmodelElements OPERATION of the AasSubmodelElements class as defined in IDTA-01001-3-0.

## AasSubmodelElements/PROPERTY

 [http://admin-shell.io/aas/3/0/AasSubmodelElements/PROPERTY](http://admin-shell.io/aas/3/0/AasSubmodelElements/PROPERTY) The AasSubmodelElements PROPERTY of the AasSubmodelElements class as defined in IDTA-01001-3-0.

## AasSubmodelElements/RANGE

 [http://admin-shell.io/aas/3/0/AasSubmodelElements/RANGE](http://admin-shell.io/aas/3/0/AasSubmodelElements/RANGE) The AasSubmodelElements RANGE of the AasSubmodelElements class as defined in IDTA-01001-3-0.

## AasSubmodelElements/REFERENCE_ELEMENT

 [http://admin-shell.io/aas/3/0/AasSubmodelElements/REFERENCE_ELEMENT](http://admin-shell.io/aas/3/0/AasSubmodelElements/REFERENCE_ELEMENT) The AasSubmodelElements REFERENCE_ELEMENT of the AasSubmodelElements class as defined in IDTA-01001-3-0.

## AasSubmodelElements/RELATIONSHIP_ELEMENT

 [http://admin-shell.io/aas/3/0/AasSubmodelElements/RELATIONSHIPT_ELEMENT](http://admin-shell.io/aas/3/0/AasSubmodelElements/RELATIONSHIPT_ELEMENT) The AasSubmodelElements RELATIONSHIPT_ELEMENT of the AasSubmodelElements class as defined in IDTA-01001-3-0.

## AasSubmodelElements/SUBMODEL_ELEMENT_COLLECTION
 Collection of Submodel Elements

 [http://admin-shell.io/aas/3/0/AasSubmodelElements/SUBMODEL_ELEMENT_COLLECTION](http://admin-shell.io/aas/3/0/AasSubmodelElements/SUBMODEL_ELEMENT_COLLECTION) The AasSubmodelElements SUBMODEL_ELEMENT_COLLECTION of the AasSubmodelElements class as defined in IDTA-01001-3-0.
 
## AasSubmodelElements/SUBMODEL_ELEMENT_LIST
 List of set of Submodel Elements

 [http://admin-shell.io/aas/3/0/AasSubmodelElements/SUBMODEL_ELEMENT_LIST](http://admin-shell.io/aas/3/0/AasSubmodelElements/SUBMODEL_ELEMENT_LIST) The AasSubmodelElements SUBMODEL_ELEMENT_LIST of the AasSubmodelElements class as defined in IDTA-01001-3-0.

## AasSubmodelElements/SUBMODEL_ELEMENT

 [http://admin-shell.io/aas/3/0/AasSubmodelElements/SUBMODEL_ELEMENT](http://admin-shell.io/aas/3/0/AasSubmodelElements/SUBMODEL_ELEMENT) The AasSubmodelElements SUBMODEL_ELEMENT of the AasSubmodelElements class as defined in IDTA-01001-3-0.

```

## Reference
 Reference to either a model element of the same or another AAs or to an external entity. A reference is an ordered list of keys, each key referencing an element. The complete list of keys may for example be concatenated to a path that then gives unique access to an element or entity.

 [http://admin-shell.io/aas/3/0/Reference](http://admin-shell.io/aas/3/0/Reference) The Reference class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

## Reference/key
 Unique reference in its name space.

 [https://admin-shell.io/aas/3/0/Reference/key](https://admin-shell.io/aas/3/0/Reference/key) The attribute key of the Reference class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

> 

Reviewer: ```suggestion
## Reference/referredSemanticId
 Expected semantic ID of the referenced model element (Reference/type=ModelReference); there typically is no semantic ID for for the referenced object of external references (Reference/type=ExternalReference).

 [https://admin-shell.io/aas/3/0/Reference/key](https://admin-shell.io/aas/3/0/Reference/referredSemanticId) The attribute referredSemanticId of the Reference class as defined in IDTA-01001-3-0.

## Reference/type
 Type of the reference
Denotes whether reference is an external reference or a model reference

 [https://admin-shell.io/aas/3/0/Reference/type](https://admin-shell.io/aas/3/0/Reference/type) The attribute type of the Reference class as defined in IDTA-01001-3-0.

```

## ReferenceElement
 A reference element is a data element that defines a logical reference to another element within the same or another AAS or a reference to an external object or entity.

 [http://admin-shell.io/aas/3/0/ReferenceElement](http://admin-shell.io/aas/3/0/ReferenceElement) The ReferenceElement class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

> 

Reviewer: ```suggestion
## ReferenceElement/value
 External reference to an external object or entity or a logical reference to another element within the same or another Asset Administration Shell (i.e. a model reference to a Referable)

 [http://admin-shell.io/aas/3/0/ReferenceElement/value](http://admin-shell.io/aas/3/0/ReferenceElement/value) The attribute value of the ReferenceElement class as defined in IDTA-01001-3-0.

```

## RelationshipElement

 [http://admin-shell.io/aas/3/0/RelationshipElement](http://admin-shell.io/aas/3/0/RelationshipElement) The RelationshipElement class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

## RelationshipElement/first
 First element in the relationship taking the role of the subject.

 [http://admin-shell.io/aas/3/0/RelationshipElement/first](http://admin-shell.io/aas/3/0/RelationshipElement/first) The attribute first of the RelationshipElement class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

## RelationshipElement/second
 Second element in the relationship taking the role of the object.

 [http://admin-shell.io/aas/3/0/RelationshipElement/second](http://admin-shell.io/aas/3/0/RelationshipElement/second) The attribute second of the RelationshipElement class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

> 

Reviewer: ```suggestion

## Resource
 Resource represents an address to a file (a locator). The value is a URI that can represent an absolute or relative path.

 [http://admin-shell.io/aas/3/0/Resource](http://admin-shell.io/aas/3/0/Resource) The Resource class as defined in IDTA-01001-3-0.

## Resource/contentType
 Content type of the content of the file

 The content type states which file extensions the file can have.

 [http://admin-shell.io/aas/3/0/Resource/contentType](http://admin-shell.io/aas/3/0/Resource/contentType) The attribute contentType of the Resource class as defined in IDTA-01001-3-0.
 
## Resource/path
 Path and name of the resource (with file extension)
 
 The path can be absolute or relative.

 [http://admin-shell.io/aas/3/0/Resource/path](http://admin-shell.io/aas/3/0/Resource/path) The attribute path of the Resource class as defined in IDTA-01001-3-0.
 
## SpecificAssetId
 A specific asset ID describes a generic supplementary identifying attribute of the asset. The specific asset ID is not necessarily globally unique.

 [http://admin-shell.io/aas/3/0/SpecificAssetId](http://admin-shell.io/aas/3/0/SpecificAssetId) The SpecificAssetId class as defined in IDTA-01001-3-0.
 
## SpecificAssetId/name
 Name of the asset identifier.

 [http://admin-shell.io/aas/3/0/SpecificAssetId/name](http://admin-shell.io/aas/3/0/SpecificAssetId/name) The attribute name  of the SpecificAsetId class as defined in IDTA-01001-3-0.
 
## SpecificAssetId/externalSubjectId
 vThe unique ID of the (external) subject the specific asset ID value belongs to or has meaning to.

 [http://admin-shell.io/aas/3/0/SpecificAssetId/externalSubjectId](http://admin-shell.io/aas/3/0/SpecificAssetId/externalSubjectId) The attribute externalSubjectId  of the SpecificAsetId class as defined in IDTA-01001-3-0.

## SpecificAssetId/value
 value of the asset identifier.

 [http://admin-shell.io/aas/3/0/SpecificAssetId/value](http://admin-shell.io/aas/3/0/SpecificAssetId/value) The attribute value  of the SpecificAsetId class as defined in IDTA-01001-3-0.

```

## Submodel
 A Submodel defines a specific aspect of the asset represented by the AAS. A submodel is used to structure the virtual representation and technical functionality of an Administration Shell into distinguishable parts. Each submodel refers to a well-defined domain or subject matter. Submodels can become standardized and thus become submodels types. Submodels can have different life-cycles.

 [http://admin-shell.io/aas/3/0/Submodel](http://admin-shell.io/aas/3/0/Submodel) The Submodel class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

> ## Submodel/submodelElement
>  A submodel consists of zero or more submodel elements.
> 
>  [http://admin-shell.io/aas/3/0/Submodel/submodelElement](http://admin-shell.io/aas/3/0/Submodel/submodelElement) The attribute submodelElement of the Submodel class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

Reviewer: ```suggestion
## Submodel/submodelElements
 A submodel consists of zero or more submodel elements.

 [http://admin-shell.io/aas/3/0/Submodel/submodelElements](http://admin-shell.io/aas/3/0/Submodel/submodelElements) The attribute submodelElement of the Submodel class as defined in IDTA-01001-3-0.
```

## SubmodelElement
 A submodel element is an element suitable for the description and differentiation of assets.

 [http://admin-shell.io/aas/3/0/SubmodelElement](http://admin-shell.io/aas/3/0/SubmodelElement) The SubmodelElement class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

## SubmodelElementCollection
 A submodel element collection is a set or list of submodel elements.

 [http://admin-shell.io/aas/3/0/SubmodelElementCollection](http://admin-shell.io/aas/3/0/SubmodelElementCollection) The SubmodelElementCollection class as defined in Details of the 'Asset Administration Shell - Part 1 - Version 3.0'.

> ## SubmodelElementCollection/allowDuplicates
>  If allowDuplicates=true then it is allowed that the collection contains the same element several times. Default = false
> 
>  [http://admin-shell.io/aas/3/0/SubmodelElementCollection/allowDuplicates](http://admin-shell.io/aas/3/0/SubmodelElementCollection/allowDuplicates) The attribute allowDuplicates of the SubmodelElementCollection class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.
> 
> ## SubmodelElementCollection/ordered
>  If ordered=false then the elements in the property collection are not ordered. If ordered=true then the elements in the collection are ordered. Default = false
> 
>  [http://admin-shell.io/aas/3/0/SubmodelElementCollection/ordered](http://admin-shell.io/aas/3/0/SubmodelElementCollection/ordered) The attribute ordered of the SubmodelElementCollection class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

Reviewer: ```suggestion

```

## SubmodelElementCollection/value
 Submodel element contained in the collection.

 [https://admin-shell.io/aas/3/0/SubmodelElementCollection/value](https://admin-shell.io/aas/3/0/SubmodelElementCollection/value)The attribute value of the SubmodelElementCollection class as defined in 'Details of the Asset Administration Shell - Part 1 - Version 3.0'.

> 

Reviewer: ```suggestion
## SubmodelElementList
 List or set of submodel elements.

 [http://admin-shell.io/aas/3/0/SubmodelElementListhttp://admin-shell.io/aas/3/0/SubmodelElementList) The SubmodelElementList class as defined in IDTA-01001-3-0.
 
## SubmodelElementList/orderRelevant
 Defines whether order in list is relevant. If orderRelevant = false, the list represents a set or a bag.

Default: True

 [http://admin-shell.io/aas/3/0/SubmodelElementList/orderRelevant](http://admin-shell.io/aas/3/0/SubmodelElementList/orderRelevant) The attribute orderRelevant of the SubmodelElementList class as defined in IDTA-01001-3-0.

## SubmodelElementList/semanticIdListElement
 Semantic ID which the submodel elements contained in the list match

 [http://admin-shell.io/aas/3/0/SubmodelElementList/semanticIdListElement](http://admin-shell.io/aas/3/0/SubmodelElementList/semanticIdListElement) The attribute semanticIdListElement of the SubmodelElementList class as defined in IDTA-01001-3-0.
 
## SubmodelElementList/typeValueListElement
 The submodel element type of the submodel elements contained in the list

 [http://admin-shell.io/aas/3/0/SubmodelElementList/typeValueListElement](http://admin-shell.io/aas/3/0/SubmodelElementList/typeValueListElement) The attribute typeValueListElement of the SubmodelElementList class as defined in IDTA-01001-3-0.

## SubmodelElementList/value
 Submodel elements contained in the list.

 [https://admin-shell.io/aas/3/0/SubmodelElementList/value](https://admin-shell.io/aas/3/0/SubmodelElementList/value)The attribute value of the SubmodelElementList class as defined in IDTA-01001-3-0.

## SubmodelElementList/valueTypeListElement
 The submodel element type of the submodel elements contained in the list

 [http://admin-shell.io/aas/3/0/SubmodelElementList/valueTypeListElement](http://admin-shell.io/aas/3/0/SubmodelElementList/valueTypeListElement) The attribute valueTypeListElement of the SubmodelElementList class as defined in IDTA-01001-3-0.

```

## DataSpecificationIEC61360/valueList

 [https://admin-shell.io/DataSpecificationTemplates/DataSpecificationIEC61360/3/0/DataSpecificationIEC61360/valueList](https://admin-shell.io/DataSpecificationTemplates/DataSpecificationIEC61360/3/0/DataSpecificationIEC61360/valueList) The attribute valueList of the DataSpecificationIEC61360 class as defined in IDTA-01003-a-3-0.

## DataSpecificationIEC61360/valueId
 The Type 'ValueList' lists all the allowed values for a concept description for which the allowed values are listed in an enumeration. The value list is a set of value reference pairs.

 [http://admin-shell.io/aas/3/0/DataSpecificationIEC61360/valueId](http://admin-shell.io/aas/3/0/DataSpecificationIEC61360/valueId) The attribute valueId of the DataSpecificationIEC61360 class as defined in DTA-01003-a-3-0.

## DataTypeIEC61360
 Enumeration of all IEC 61360 defined data types.

 [https://admin-shell.io/DataSpecificationTemplates/DataSpecificationIEC61360/3/0/DataTypeIEC61360](https://admin-shell.io/DataSpecificationTemplates/DataSpecificationIEC61360/3/0/DataTypeIEC61360) The DataTypeIEC61360 class as defined in IDTA-01003-a-3-0.

> ## DataTypeIEC61360/BOOLEAN_IEC6360_DATATYPE

Reviewer: IEC61360 not IEC6360: for all values 

Why do we add _IEC61360 and DATATYPE at all? The enumeration vale is just BOOLEAN?

https://admin-shell-io.github.io/aas-specs-antora/IDTA-01003-a/v3.1/specification.html#_enumeration_data_type_iec61360

## DataTypeIEC61360/URL_IEC6360_DATATYPE

 [http://admin-shell.io/aas/3/0/DataTypeIEC61360/URL_IEC6360_DATATYPE](http://admin-shell.io/aas/3/0/DataTypeIEC61360/URL_IEC6360_DATATYPE) The DataTypeIEC61360 URL_IEC6360_DATATYPE of the DataTypeIEC61360 class as defined in DTA-01003-a-3-0.

## Entity
 An entity is a submodel element that is used to model entities.

 [http://admin-shell.io/aas/3/0/Entity](http://admin-shell.io/aas/3/0/Entity) The Entity class as defined in IDTA-01001-3-0.

## Entity/entityType
 Describes whether the entity is a co-managed entity or a self-managed entity.

 [http://admin-shell.io/aas/3/0/Entity/entityType](http://admin-shell.io/aas/3/0/Entity/entityType) The attribute entityType of the Entity class as defined in IDTA-01001-3-0.

## Entity/globalAssetId
 Reference to the asset the entity is representing.

 [http://admin-shell.io/aas/3/0/Entity/globalAssetId](http://admin-shell.io/aas/3/0/Entity/globalAssetId) The attribute globalAssetId of the Entity class as defined in IDTA-01001-3-0.

> ## Entity/specificAssetId

Reviewer: Add empty line

 [http://admin-shell.io/aas/3/0/EntityType](http://admin-shell.io/aas/3/0/EntityType) The EntityType class as defined in IDTA-01001-3-0.

## EntityType/CoManagedEntity
 For co-managed entities there is no separate AAS. Co-managed entities need to be part of a self-managed entity.

 [http://admin-shell.io/aas/3/0/EntityType/CO_MANAGED_ENTITY](http://admin-shell.io/aas/3/0/EntityType/CO_MANAGED_ENTITY) The NamedIndividual CO_MANAGED_ENTITY of the EntityType class as defined in IDTA-01001-3-0.

## EntityType/SelfManagedEntity
 Self-Managed Entities have their own AAS but can be part of the bill of material of a composite self-managed entity. The asset of an I4.0 Component is a self-managed entity per definition.

 [http://admin-shell.io/aas/3/0/EntityType/SELF_MANAGED_ENTITY](http://admin-shell.io/aas/3/0/EntityType/SELF_MANAGED_ENTITY) The NamedIndividual SELF_MANAGED_ENTITY of the EntityType class as defined in IDTA-01001-3-0.
## Extension
 Single extension of an element.

 [https://admin-shell.io/aas/3/0/Extension](https://admin-shell.io/aas/3/0/Extension) The Extension class as defined in IDTA-01001-3-0.

> ## Extension/name

Reviewer: Add empty line

## Identifiable/id
 The globally unique identification of the element.

 [https://admin-shell.io/aas/3/0/Identifiable/id](https://admin-shell.io/aas/3/0/Identifiable/id) The attribute identification of the Identifiable class as defined in IDTA-01001-3-0.

## Identifier
 Used to uniquely identify an entity by using an identifier.

 [http://admin-shell.io/aas/3/0/Identifier](http://admin-shell.io/aas/3/0/Identifier) The Identifier class as defined in IDTA-01001-3-0.

## Identifier/id
 A globally unique identifier which might not be a URI. Its type is defined in idType.

 [http://admin-shell.io/aas/3/0/Identifier/id](http://admin-shell.io/aas/3/0/Identifier/id) The attribute id of the Identifier class as defined in IDTA-01001-3-0.

## Identifier/idType
 Type of the Identifier, e.g. IRI, IRDI etc. The supported Identifier types are defined in the enumeration 'IdentifierType'.

 [http://admin-shell.io/aas/3/0/Identifier/idType](http://admin-shell.io/aas/3/0/Identifier/idType) The attribute idType of the Identifier class as defined in IDTA-01001-3-0.

> ## IdentifierType

Reviewer: IdentifierType and its values not part of spec any longer:  remove

 Enumeration of different types of Identifiers for global identification

 [http://admin-shell.io/aas/3/0/IdentifierType](http://admin-shell.io/aas/3/0/IdentifierType) The IdentifierType class as defined in IDTA-01001-3-0.

## IdentifierType/CUSTOM
 Custom identifiers like GUIDs (globally unique Identifiers)

 [http://admin-shell.io/aas/3/0/IdentifierType/CUSTOM](http://admin-shell.io/aas/3/0/IdentifierType/CUSTOM) The IdentifierType CUSTOM of the IdentifierType class as defined in IDTA-01001-3-0.

## IdentifierType/IRDI
 IRDI according to ISO29002-5 as an Identifier scheme for properties and classifications.

 [http://admin-shell.io/aas/3/0/IdentifierType/IRDI](http://admin-shell.io/aas/3/0/IdentifierType/IRDI) The IdentifierType IRDI of the IdentifierType class as defined in IDTA-01001-3-0.

## IdentifierType/IRI
 IRI. Should only be used if unicode symbols are used that are not allowed in URI.

 [http://admin-shell.io/aas/3/0/IdentifierType/IRI](http://admin-shell.io/aas/3/0/IdentifierType/IRI) The IdentifierType IRI of the IdentifierType class as defined in IDTA-01001-3-0.

> ## IdentifierKeyValuePair

Reviewer: Not existing any longer

## KeyElement
 Enumeration of different key value types within a key. Contains KeyElements, ReferableElements, and IdentifiableElements.

 [http://admin-shell.io/aas/3/0/KeyElement](http://admin-shell.io/aas/3/0/KeyElement) The KeyElement class as defined in IDTA-01001-3-0.

## KeyElement/FRAGMENT_REFERENCE

 [http://admin-shell.io/aas/3/0/KeyElement/FRAGMENT_REFERENCE](http://admin-shell.io/aas/3/0/KeyElement/FRAGMENT_REFERENCE) The KeyElement FRAGMENT_REFERENCET of the KeyElement class as defined in IDTA-01001-3-0.

## KeyElement/GLOBAL_REFERENCE
 reference to an element not belonging to an asset administration shell

 [http://admin-shell.io/aas/3/0/KeyElement/GLOBAL_REFERENCE](http://admin-shell.io/aas/3/0/KeyElement/GLOBAL_REFERENCE) The KeyElement GLOBAL_REFERENCE of the KeyElement class as defined in IDTA-01001-3-0.

## KeyTypes
 Enumeration of different key value types within a key.

 [http://admin-shell.io/aas/3/0/KeyType](http://admin-shell.io/aas/3/0/KeyType) The KeyType class as defined in IDTA-01001-3-0.

## LevelType

>  Enumeration of different level types within a DataSpecificationIEC61360. Contains Min, Max, Nom, and Typ.

Reviewer: No enumeration any longer but attributes still there

 [https://admin-shell.io/aas/3/0/LevelType](https://admin-shell.io/aas/3/0/LevelType) The LevelType class as defined in IDTA-01003-a-3-0.

## LevelType/Min
 Min according to IEC 61360 as an Identifier scheme for minimal levels.

 [https://admin-shell.io/aas/3/0/LevelType/MIN](https://admin-shell.io/aas/3/0/LevelType/MIN) The MIN of the LevelType class as defined in IDTA-01001-3-0.

## LevelType/MAX
 MAX according to IEC 61360 as an Identifier scheme for maximal levels.

 [https://admin-shell.io/aas/3/0/LevelType/MAX](https://admin-shell.io/aas/3/0/LevelType/MAX) The MAX of the LevelType class as defined in IDTA-01001-3-0.

## LevelType/NOM
 NOM according to IEC 61360 as an Identifier scheme for nominal levels.

 [https://admin-shell.io/aas/3/0/LevelType/NOM](https://admin-shell.io/aas/3/0/LevelType/NOM) The NOM of the LevelType class as defined in IDTA-01001-3-0.

## LevelType/TYP
 TYP according to IEC 61360 as an Identifier scheme for typical levels.

 [https://admin-shell.io/aas/3/0/LevelType/TYP](https://admin-shell.io/aas/3/0/LevelType/TYP) The TYP of the LevelType class as defined in IDTA-01001-3-0.

> ## LocalKeyType

Reviewer: Not existing any longer

 [http://admin-shell.io/aas/3/0/LocalKeyType](http://admin-shell.io/aas/3/0/LocalKeyType) The LocalKeyType class as defined in IDTA-01001-3-0.

## LocalKeyType/FRAGMENT_ID
 Identifier of a fragment within a file

 [http://admin-shell.io/aas/3/0/LocalKeyType/FRAGMENT_ID](http://admin-shell.io/aas/3/0/LocalKeyType/FRAGMENT_ID) The LocalKeyType FRAGMENT_ID of the LocalKeyType class as defined in IDTA-01001-3-0.

## LocalKeyType/IDSHORT
 idShort of a referable element

 [http://admin-shell.io/aas/3/0/LocalKeyType/IDSHORT](http://admin-shell.io/aas/3/0/LocalKeyType/IDSHORT) The LocalKeyType IDSHORT of the LocalKeyType class as defined in IDTA-01001-3-0.

## ModellingKind
 Enumeration for denoting whether an element is a type or an instance.

 [https://admin-shell.io/aas/3/0/ModellingKind](https://admin-shell.io/aas/3/0/ModellingKind) The ModellingKind class as defined IDTA-01001-3-0.

## ModelingKind/Instance
 Concrete, clearly identifiable component of a certain template.

 [http://admin-shell.io/aas/3/0/ModelingKind/INSTANCE](http://admin-shell.io/aas/3/0/ModelingKind/INSTANCE) The NamedIndividual INSTANCE of the ModelingKind class as defined in IDTA-01001-3-0.

> ## ModelingKind/TEMPLATE

Reviewer: Instance not in capital letters

 [http://admin-shell.io/aas/3/0/OperationVariable/value](http://admin-shell.io/aas/3/0/OperationVariable/value) The attribute value of the OperationVariable class as defined in IDTA-01001-3-0.

## Property
 A property is a data element that has a single value.

 [http://admin-shell.io/aas/3/0/Property](http://admin-shell.io/aas/3/0/Property) The Property class as defined in IDTA-01001-3-0.

## Property/value
 The value of the property instance.

 [https://admin-shell.io/aas/3/0/Property/value](https://admin-shell.io/aas/3/0/Property/value) The attribute value of the Property class as defined in IDTA-01001-3-0.

## Property/valueId
 Reference to the global unique id of a coded value.

 [https://admin-shell.io/aas/3/0/Property/valueId](https://admin-shell.io/aas/3/0/Property/valueId) The attribute valueId of the Property class as defined in IDTA-01001-3-0.

## Property/valueType
 Data type of the value attribute.

 [https://admin-shell.io/aas/3/0/Property/valueType](https://admin-shell.io/aas/3/0/Property/valueType) The attribute valueType of the Property class as defined in IDTA-01001-3-0. 

> ## Qualifiable

Reviewer: Add empty line
